src/python/setup.py

```diff
@@ -32,15 +30,27 @@
     author="The tensorflow cloud authors",
     author_email="tensorflow-cloud@google.com",
     license="Apache License 2.0",
-    install_requires=["docker", "google-api-python-client", "google-cloud-storage",],
-    extras_require={"tests": ["pytest", "flake8", "mock"],},
+    extras_require={"tests": dependencies.make_required_test_packages()},
+    include_package_data=True,
+    install_requires=dependencies.make_required_install_packages(),
     classifiers=[
-        "Programming Language :: Python",
+        "Development Status :: 4 - Beta",
```

Comment:
What does this mean?

Reply:
good catch, removing this since there are multiple modules under this package package level deployment status won't be meaningful here.